docs/en/sql-reference/functions/array-functions.md

```diff
@@ -2157,6 +2157,41 @@ Result:
 └───────────────────────────────────────────────┘
 ```
 
+## arrayPrAUC
+
+Calculate AUC (Area Under the Curve) for the Precision Recall curve.
+
+**Syntax**
+
+``` sql
+arrayPrAUC(arr_scores, arr_labels)
+```
+
+**Arguments**
+
+- `arr_scores` — scores prediction model gives.
+- `arr_labels` — labels of samples, usually 1 for positive sample and 0 for negative sample.
+
+**Returned value**
+
+Returns PR-AUC value with type Float64.
+
+**Example**
+
+Query:
+
+``` sql
+select arrayPrAUC([0.1, 0.4, 0.35, 0.8], [0, 0, 1, 1]);
+```
+
+Result:
+
+``` text
+┌─arrayAUC([0.1, 0.4, 0.35, 0.8], [0, 0, 1, 1])─┐
```

Comment:
Why is it arrayAUC in the query result if we have arrayPrAUC?

Reply:
It should be arrayPrAUC, I'll fix it